hack/jenkins/build_iso.sh

```diff
@@ -40,12 +40,16 @@ else
 	export ISO_BUCKET
 fi
 
-make release-iso
+make release-iso | tee iso-logs.txt
 # Abort with error message if above command failed
 ec=$?
 if [ $ec -gt 0 ]; then
 	if [ "$release" = false ]; then
-		gh pr comment ${ghprbPullId} --body "Hi ${ghprbPullAuthorLoginMention}, building a new ISO failed, please try again."
+		err=$(tail -50 iso-logs.txt)
+		gh pr comment ${ghprbPullId} --body "Hi ${ghprbPullAuthorLoginMention}, building a new ISO failed, with the error below:
+		
+		${err}
```

Comment:
how about we make this
<details>
<pre>
${err}

</pre>
</details>

Reply:
my thought here is usually the error is printed right at the end of the file. i wanted to display the error without too much other cruft that can overwhelms us with unnecessary info